plugins/outputs/kairosdb/kairosdb.go

```diff
@@ -0,0 +1,287 @@
+package kairosdb
+
+import (
+	"fmt"
+	"log"
+	"net"
+	"net/url"
+	"regexp"
+	"sort"
+	"strconv"
+	"strings"
+
+	"github.com/influxdata/telegraf"
+	"github.com/influxdata/telegraf/plugins/outputs"
+)
+
+var (
+	allowedChars = regexp.MustCompile(`[^a-zA-Z0-9-_./\p{L}]`)
+	hypenChars   = strings.NewReplacer(
+		"@", "-",
+		"*", "-",
+		`%`, "-",
+		"#", "-",
+		"$", "-")
+	defaultHttpPath  = "/api/v1/datapoints"
+	defaultSeperator = "_"
+)
+
+type KairosDB struct {
+	Prefix string
+
+	Host string
+	Port int
```

Comment:
Please add toml annotations also for the parameters below:
```suggestion
	Prefix string `toml:"prefix"`

	Host string `toml:"host"`
	Port int `toml:"port"`
```